src/molecules/gv-card-full.js

```diff
@@ -176,12 +185,16 @@ export class GvCardFull extends ItemResource(LitElement) {
 
   render() {
     const title = getTitle(this._item);
+    const owner = getOwner(this._item);
     const classes = { error: this._error || this._empty, card: true };
     return html`<div class="${classMap(classes)}" title="${title}" @click="${this._onClick}">
       <div class="${classMap({ skeleton: this._skeleton })}">
         <div class="${classMap({ image: true })}">${this._renderImage()}</div>
         <div class="content">
           <div class="${classMap({ title: true })}">${title}</div>
+          ${owner != null && owner.trim().length > 0
+            ? html`<div class="owner"><gv-icon shape="general:user" size="8px"></gv-icon>${owner}</div>`
```

Comment:
Maybe some extra style is needed to align the icon

Reply:
👍 👓

Reply:
Thanks
better ? 
![image](https://user-images.githubusercontent.com/4974420/142989536-e24fca6e-2c0f-46ff-a649-c4489a596892.png)